Dolapo47/Auto-Mart
# Ft dashboard #165957600

#### What does this PR do?
- A summary of what this task is about
Create the dashboard page, where users can access cars on sale
#### Description of Task to be completed?
- A list of the major tasks you did to complete this feature/bug/chore
-Create the header
-Create the navbar to house the create a new sale link
-Create a view to house the individual cars up for sale

#### How should this be manually tested?
- Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
When a user has successfully logged into the app, the user will be directed to the dashboard where he can view the cars for sale

#### What are the relevant pivotal tracker stories?
- PT Story link - (format - `[#165957600](https://www.pivotaltracker.com/n/projects/2346179)`)
